### PR TITLE
adding Microsoft-Support.md

### PR DESCRIPTION
**Pull request by HongGit**
adding Microsoft-Support.md

### CODE REVIEW

File: Documentation/Microsoft.Support.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Microsoft Support
+
+This document details Microsoft Support pollicy for CoreWCF.
+
+Runtime Version | Support dependency duration |
+--- | --- |
+.NET Framework 4.x | The specified version of .NET Framework and ASP.NET Core 2.1 |
+Net Core 3.1 | .NET 3.1 LTS - DEcembmer 3, 2022 |
```
Comment (mconnew) [inline Documentation/Microsoft.Support.md:8]: December has weird casing and incorrect spelling.

File: Documentation/Microsoft.Support.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Microsoft Support
+
+This document details Microsoft Support pollicy for CoreWCF.
+
+Runtime Version | Support dependency duration |
+--- | --- |
+.NET Framework 4.x | The specified version of .NET Framework and ASP.NET Core 2.1 |
```
Comment (mconnew) [inline Documentation/Microsoft.Support.md:7]: Is it worth making ".NET Framework" and "ASP.NET Core 2.1" link to https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework and https://dotnet.microsoft.com/en-us/platform/support/policy/aspnetcore-2.1 respectively?

Comment (HongGit) [reply Documentation/Microsoft.Support.md:7]: make sense

File: Documentation/Microsoft.Support.md (hunk 1 above)
Comment (HongGit) [reply Documentation/Microsoft.Support.md:8]: Thanks for catching that!

**Comment by samsp-msft** [comment]
I moved more of the content from the blog into this, and have removed it from the roadmap md.

**Pull request merged**